File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/vorkath/RANGE_POTION.java
package net.runelite.client.plugins.microbot.vorkath;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.val;

@Getter
@RequiredArgsConstructor
public enum RANGE_POTION {
    DIVINE_RANGING_POTION("Ranging Potion"),
    DIVINE_BASTION_POTION("Bastion Potion");

    private final String potionName;

    @Override
    public String toString() { return potionName; }
}

File: runelite-client/src/main/java/net/runelite/client/plugins/jrPlugins/autoVorkath/RANGE_POTION.kt
package net.runelite.client.plugins.jrPlugins.autoVorkath

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class RANGE_POTION(val potionName: String) {
    DIVINE_RANGING_POTION("Ranging Potion"),
    DIVINE_BASTION_POTION("Bastion Potion");

    override fun toString(): String { return potionName }
}
